.github/workflows/checks.yml

```diff
@@ -43,21 +46,8 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
+      - name: Compile
+        run: npm run build
+
       - name: Run e2e tests
         run: npm run test:e2e
-
-      - name: Upload Screenshots
```

Comment:
is it ever worth to keep screenshots, or not really?

Reply:
Not sure, until screenshot tests are introduce I'm not sure it's needed although I imagine it actually doesn't add too much time to the test run. It's your call